diff --git a/app/src/main/java/com/example/dummy_quiz_using_agent/MainActivity.kt b/app/src/main/java/com/example/dummy_quiz_using_agent/MainActivity.kt
index 3ebcf2f..ff515e2 100644
--- a/app/src/main/java/com/example/dummy_quiz_using_agent/MainActivity.kt
+++ b/app/src/main/java/com/example/dummy_quiz_using_agent/MainActivity.kt
@@ -18,7 +18,7 @@ class MainActivity : ComponentActivity() {
 
     private val repository: QuizRepository by lazy {
         QuizRepository(
-            geminiService = GeminiService(apiKey = "wrong API KEy s")
+            geminiService = GeminiService(apiKey = "wrong API KEy use it from BuildConfig.GEMINI_API_KEY")
         )
     }
 
